.protolint.yaml

```diff
@@ -0,0 +1,4 @@
+lint:
+  rules_option:
+    max_line_length:
+      max_chars: 100
```

Comment:
the line:

```
option go_package = "github.com/fornellas/resonance/internal/host/agent_server_grpc/proto";
```

breaks the default=80, so I've bumped it to 100.

Reply:
Ok